redirects.js

```diff
@@ -171,7 +171,12 @@ const redirects = [
   },
   {
     source: '/docs/rpc/objects',
-    destination: '/docs/interacting-with-geth/rpc/objects',
+    destination: '/',
```

Comment:
Should this be `/docs` so that they are redirected within the context this page would have been? Or is this a better UX?

Reply:
I would agree w/ `/docs`